tests/mock.data.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 - 2024 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: LicenseRef-w3c-3-clause-bsd-license-2008 OR LicenseRef-w3c-test-suite-license-2023
+ */
+
+import {createRequire} from 'node:module';
+import {klona} from 'klona';
+const require = createRequire(import.meta.url);
+const validVc = require('./validVc.json');
+
+// copies a validVc
+export const createRequestBody = ({issuer, vc = validVc}) => {
+  const {settings: {id, options}} = issuer;
+  const credential = klona(vc);
+  if(credential.issuer !== null && typeof credential.issuer === 'object') {
+    if(!('id' in credential.issuer && credential.issuer.id === null)) {
+      credential.issuer.id = credential.issuer?.id || id;
+    }
+  } else {
+    if(!('issuer' in credential && credential.issuer === null)) {
+      credential.issuer = credential?.issuer || id;
+    }
+  }
+  return {
+    credential,
+    options
+  };
+};
+
+export const createVerifyRequestBody = ({vc}) => {
+  const body = {
+    verifiableCredential: vc,
+    options: {
+      checks: ['proof'],
```

Comment:
Ok so the `checks` thing is not a VC-API option: https://w3c-ccg.github.io/vc-api/#verify-credential

So we should be looking for implementation specific options and passing them to a specific endpoint's such as here: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/1f5c835edf8607d88d6e10776ebf1791828902fa/assertions.js#L36-L47

Reply:
yes, I should remove this. This should probably also be removed from vcdm 2.0. My question would be if there's implementations who currently rely on this? I believe we had made the decision to leave it in case some implementations rely on it and others can just ignore it.

Reply:
@PatStLouis DB lists it as an option: https://github.com/w3c/vc-test-suite-implementations/blob/fddda745c49ef532f52cc751cb4718d1430d4e9d/implementations/DigitalBazaar.json#L98-L103

so perhaps you should run with checks for a valid verifier tests and record results and then run it with only the implementation specific verifier options and see what happens? If an implementer needs checks then they should be informed and a PR should be put out to add that option to their implementations' verifier endpoints.

Reply:
I used the implementation provided options instead as suggested.